### PR TITLE
chore: disable telemetry in our private network

### PR DESCRIPTION
**Pull request by Rqnsom**
- Nodes shouldn't try to access anything outside of their temp folders, so copy the logging.config to private network directories
- Small readme update
- Index private network nodes and wallets to start from zero

### CODE REVIEW

**Comment by ljedrz** [comment]
There seem to be some other changes bundled in this PR; they should be documented, at least in the description.

File: tools/logging.config [hunk 1]
```diff
@@ -0,0 +1 @@
+{"Enable":false,"SendToLog":false,"URI":"","Name":"Primary","GUID":"06411ccb-ccc0-4ba6-8c9b-bc67efac2e5f","FilePath":"","UserName":"","Password":"","MinLogLevel":3,"ReportHistoryLevel":3}
```
Comment (ljedrz) [inline tools/logging.config:1]: can the GUID be hardcoded here?

Comment (Rqnsom) [reply tools/logging.config:1]: I've just tested a blank field for the GUID, and it works. Since we are disabling, it doesn't matter. Let's leave it blank

Comment (Rqnsom) [reply tools/logging.config]: Done

**Review by ljedrz — APPROVED** [review]
LGTM

**Pull request merged**